sqs/Module.md

```diff
@@ -1,83 +1,25 @@
-[![Build Status](https://travis-ci.org/ballerina-platform/module-ballerinax-aws.sqs.svg?branch=master)](https://travis-ci.org/ballerina-platform/module-ballerinax-aws.sqs)
+## Overview
+Ballerina connector for Amazon SQS is connecting the Amazon SQS API via Ballerina language easily. It provides capability to perform operations related to queues and messages.
 
-# Ballerina Amazon SQS Connector
+This module supports [Amazon SQS API](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/welcome.html) version 2012-11-05.
 
-Amazon SQS Connector allows you to connect to the Amazon Simple Queue Service (SQS) via REST API from Ballerina.
+## Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+* Create [AWS account](https://aws.amazon.com)
```

Comment:
Make "Create" also a part of the link text.

Reply:
That is not how it is written in other connector as well